### PR TITLE
Add support for JavaScript events

### LINKED ISSUE #3: Dispatch formset events
Hi,

It would be nice to include some events related to adding and deleting the formset. This is done in django too.

https://github.com/django/django/blob/main/django/contrib/admin/static/admin/js/inlines.js#L91-L96
https://github.com/django/django/blob/main/django/contrib/admin/static/admin/js/inlines.js#L138-L142

If you prefer I can do a PR of it.

### PR DESCRIPTION
**Pull request by CleitonDeLima**
Resolve #3

### CODE REVIEW

**Comment by tiesjan** [comment]
Great suggestion, @CleitonDeLima, and thanks for this PR! Although I didn't have the need for it yet, the thought had crossed my mind. When I have some more time, I will have a closer look at it. Thanks for your patience! 🙏🏻

File: convenient_formsets/static/convenient_formsets/convenient_formsets.js [hunk 1]
```diff
@@ -258,6 +258,14 @@ const ConvenientFormset = function(options) {
         // Update the form indexes and the management form
         updateFormIndexes();
         updateManagementForm();
+
+        // Dispatch event
+        formsetElements.formsContainer.dispatchEvent(new CustomEvent("convenientformset:added", {
```
Comment (CleitonDeLima) [inline convenient_formsets/static/convenient_formsets/convenient_formsets.js:263]:
```suggestion
        formsetElements.formsContainer.dispatchEvent(new CustomEvent('convenientformset:added', {
```

File: convenient_formsets/static/convenient_formsets/convenient_formsets.js [hunk 2]
```diff
@@ -287,6 +295,13 @@ const ConvenientFormset = function(options) {
         // Update the form indexes and the management form
         updateFormIndexes();
         updateManagementForm();
+
+        // Dispatch event
+        document.dispatchEvent(new CustomEvent("convenientformset:removed", {
```
Comment (CleitonDeLima) [inline convenient_formsets/static/convenient_formsets/convenient_formsets.js:300]:
```suggestion
        document.dispatchEvent(new CustomEvent('convenientformset:removed', {
```

**Review by CleitonDeLima — COMMENTED** [review]
Fix eslint

**Comment by tiesjan** [comment]
Thanks again for your patience, very nice work! 💪🏻

I have cleaned up the tests a bit and added some documentation.

**Pull request merged**